rule/file-header.go

```diff
@@ -10,73 +9,57 @@ import (
 // FileHeaderRule lints given else constructs.
 type FileHeaderRule struct{}
 
+var (
+	multiRegexp  = regexp.MustCompile("^/\\*")
+	singleRegexp = regexp.MustCompile("^//")
+)
+
 // Apply applies the rule to given file.
 func (r *FileHeaderRule) Apply(file *lint.File, arguments lint.Arguments) []lint.Failure {
-	var failures []lint.Failure
-
 	header, ok := arguments[0].(string)
```

Comment:
This line panics (index out of range) if the `arguments` slice is empty (i.e. when the rule is not configured appropriately)  
We need to check the length of the slice before accessing it. Something like
```go
if len(arguments) != 1 {
	panic(`invalid configuration for "file-header" rule`)
}
```

Reply:
Done